### PR TITLE
Srt html styling

### PR DESCRIPTION
**Pull request by justin-taylor**
This PR resurrects https://github.com/asticode/go-astisub/pull/96 The changes from the PR have been re-applied and made to work with the mast recent changes. For our use-case we often run into an srt file being uploaded with stylings that are then broken when converted to webvtt.

I believe all the comments that were apart of that PR have been addressed, but please let me know if other changes are needed.

### CODE REVIEW

File: subtitles_test.go [hunk 1]
```diff
@@ -40,6 +40,60 @@ func assertSubtitleItems(t *testing.T, i *astisub.Subtitles) {
...
+	assert.Equal(t, 1*time.Minute+26*time.Second+787*time.Millisecond, i.Items[4].StartAt)
+	assert.Equal(t, 1*time.Minute+29*time.Second+523*time.Millisecond, i.Items[4].EndAt)
+	assert.Equal(t, "[beeping]", i.Items[4].Lines[0].String())
+	assert.Equal(t, 1*time.Minute+29*time.Second+590*time.Millisecond, i.Items[5].StartAt)
+	assert.Equal(t, 1*time.Minute+31*time.Second+992*time.Millisecond, i.Items[5].EndAt)
+	assert.Equal(t, "[automated]", i.Items[5].Lines[0].String())
+	assert.Equal(t, "'The time is 7:35.'", i.Items[5].Lines[1].String())
+}
+
+func assertSRTSubtitleStyles(t *testing.T, i *astisub.Subtitles) {
```
Comment (asticode) [inline subtitles_test.go:66]: This function should be removed based on my previous comment

**Review by asticode — CHANGES_REQUESTED** [review]
Thanks for picking up this PR ❤️ 

A few changes are needed for tests 👍

**Comment by justin-taylor** [comment]
> Thanks for picking up this PR ❤️
> 
> A few changes are needed for tests 👍

All comments have been addressed.

**Comment by asticode** [comment]
Thanks for the PR! ❤️ 

Let me know whether you need a tag 👍

**Comment by justin-taylor** [comment]
> Thanks for the PR! ❤️
> 
> Let me know whether you need a tag 👍

A tag would be great! Thank you.

**Comment by asticode** [comment]
I've created a `v0.30.0` tag 👍

**Pull request merged**